Formula/libxlsxwriter.rb

```diff
@@ -0,0 +1,35 @@
+class Libxlsxwriter < Formula
+  desc "Libxlsxwriter: A C library for creating Excel XLSX files."
+  homepage "https://libxlsxwriter.github.io/"
+  url "https://github.com/jmcnamara/libxlsxwriter/archive/RELEASE_0.7.3.tar.gz"
+  sha256 "84dbbd164b440695119ded3a8e2b6cac07d0d284343ab977c88dc0418d8fcde5"
+
+  depends_on "lzlib"
+
+  def install
+    lib.mkdir
+    include.mkdir
+    system "make", "V=1"
+    system "make", "install", "INSTALL_DIR=#{prefix}", "V=1"
```

Comment:
`make` is the secondary build system for people who prefer it. The make build should really be 1 step as well but there is an error that I didn't notice until I was preparing this formula. Once it is fixed the following will be sufficient:

    system "make", "install", "INSTALL_DIR=#{prefix}", "V=1"

That will need a respin of libxlsxwriter to 0.7.4 though.